build/purger/purger.py

```diff
@@ -159,72 +148,40 @@ def purge(
 
 
 def run_user_purger(
-    api: GMatrixHttpApi,
-    global_user_activity: UserActivityInfo,
-    network_to_broadcast_rooms: Dict[str, List[RoomInfo]],
+    api: GMatrixHttpApi, global_user_activity: UserActivityInfo,
 ) -> UserActivityInfo:
     """
     The user purger mechanism finds inactive users which have been offline
-    longer than the threshold time and will kick them out of the global rooms
-    to clean up the server's database and reduce load.
+    longer than the threshold time and will deactivate them. By deactivating them
```

Comment:
I'd use `delete` instead of `deactivate`. Even though the endpoint is called `deactivate`, semantically we're deleting them.

(Similar for all the following occurrences)